packages/server/src/api/controllers/static.js

```diff
@@ -73,5 +93,17 @@ exports.serveComponentLibrary = async function(ctx) {
     )
   }
 
+  if (process.env.CLOUD) {
+    const appId = ctx.user.appId
```

Comment:
would it be better to return a 302 ? Not sure if that actually works for assets.

I think the ultimate solution is to re-route these requests before its gets here... so maybe this comment is of no use.

If we do have to serve out of this app, I suggest that we pipe a stream from S3 - rather than load from S3 into memory, the set the body.

Reply:
Yeah, this was more of an intermediary - it defeats the purpose having the node server serve this.  Setting the fetch promise to the body will stream the bytes back to the browser AFAIK, but it's still not a good solution.